### PR TITLE
Add attributes to the session cards

### LINKED ISSUE #105: Try to implement some attributes

### PR DESCRIPTION
**Pull request by MayKikuchi**
fixed issue in previous pr

closes #105

### CODE REVIEW

**Comment by MayKikuchi** [comment]
merge conflicts should be fixed now

File: src/ast/ATTRIBUTES.ts [hunk 1]
```diff
@@ -4,9 +4,11 @@ import ATTRIBUTE from "./ATTRIBUTE";
 export default class ATTRIBUTES extends NODE {
   attributes: ATTRIBUTE[] = [];
   parse() {
-    const areAttributes = this.tokenizer.checkToken(
-      "add color|add alignment|add direction"
-    );
+    const nextToken = this.tokenizer.checkNext().toLowerCase();
+    const areAttributes =
```
Comment (algobot76) [inline src/ast/ATTRIBUTES.ts:8]: I think this could be refactored as a small helper function.

File: src/ast/CREATE_DECK.ts [hunk 2]
```diff
@@ -20,7 +20,12 @@ export default class CREATE_DECK extends NODE {
   }
 
   checkForAndParseAttributes() {
-    if (this.tokenizer.checkToken("add color|add alignment|add direction")) {
+    const nextToken = this.tokenizer.checkNext().toLowerCase();
+    if (
```
Comment (algobot76) [inline src/ast/CREATE_DECK.ts:24]: same here

File: src/ast/ATTRIBUTES.ts (hunk 1 above)
Comment (MayKikuchi) [reply src/ast/ATTRIBUTES.ts]: sure

**Pull request merged**